tools/mksurfdata_esmf/gen_mksurfdata_namelist.xml

```diff
@@ -97,7 +97,7 @@
 
   <mksrf_furban>
     <entry>
-      <data_filename>/glade/p/cgd/tss/people/oleson/Dynamic_Urban_Data/GaoOneil/PCTURB_GRID/Historical/urban_properties_GaoOneil_05deg_ThreeClass_%y_c20220910.nc</data_filename>
+      <data_filename>lnd/clm2/rawdata/gao_oneill_urban/historical/urban_properties_GaoOneil_05deg_ThreeClass_%y_c20220910.nc</data_filename>
```

Comment:
@olyson @ekluzek 
A heads up: When I updated the paths in this .xml, I missed that the file names now have the `_cdf5_` label, e.g:
`urban_properties_GaoOneil_05deg_ThreeClass_%y_cdf5_c20220910.nc`

I am correcting this in #1732 which I consider the next branch to soon be merged in the ctsm5.2.mksurfdata branch.